common/interfaces/user_interface/ui_list.h

```diff
@@ -12,7 +12,10 @@
 
 #include "ui_common.h"
 
-//TODO : Add constant
+//TODO: Update count for higher coin list
+#define MAX_UI_LIST_WORDS       24
+#define MAX_UI_LIST_CHAR_LEN    16
```

Comment:
The limit of 15 on length of each entry to the list is not useful. The add coin flow lists the coin names which could be larger.

Reply:
To solve this issue we can use the following solution
- Make option_list a array of pointer instead of a double array
- use wrapper for mnemonic implementation and different API other uses of ui_list which take array of pointer as input.

Reply:
This change will be picked up in https://github.com/Cypherock/x1_wallet_firmware/pull/221